Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -542,6 +540,57 @@ - (NSInteger)getBranchViewCount:(NSString *)branchViewID {
     return count;
 }
 
+- (void)saveBranchAnalyticsData:(NSDictionary *)analyticsData {
+    if (_sessionID != nil) {
+        if (_savedAnalyticsData == nil) {
```

Comment:
also cleaner to do `if (!_savedAnalyticsData) {` - can you go through rest and update following this pattern?